tests/subsys/usb_c/mocks/mock_tester.c

```diff
@@ -0,0 +1,393 @@
+/*
+ * Copyright 2023 The Chromium OS Authors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/kernel.h>
+#include <string.h>
+#include <stddef.h>
+#include <zephyr/drivers/usb_c/usbc_pd.h>
+
+#include "mock_tester.h"
+#include "mock_tcpc.h"
+
+/**
+ * @breif Tester data
+ */
+struct mock_tester_t {
+	/* pointer to mock tcpc */
+	struct mock_tcpc_data *tcpc;
+	/* CC1 voltage state */
+	enum tc_cc_voltage_state cc1;
```

Comment:
Are these the voltages set by the UUT? If so, do they have different meaning than `uut_cc_pull`?